README.md

```diff
@@ -23,12 +23,11 @@ git submodule update --init --recursive
 
 ## Versioning
 
-The examples contained in the
-[master](https://github.com/rticommunity/rticonnextdds-examples/tree/master)
-branch of this repository have been built and tested against RTI Connext DDS
-7.3.0. If you need examples that have been built and tested against older
-versions of RTI Connext DDS, please check out the appropriate branch:
+The examples contained in this branch were built and tested against **RTI Connext
+7.4.0 EAR**. If you need examples that have been built and tested against older
+versions of RTI Connext, please check out **master** or the appropriate branch:
```

Comment:
No, this paragraph should NOT be changed as we are not merging 7.4.0 back to master as 7.4.0 is not LTS release.